skopt/optimizer/gp.py

```diff
@@ -15,8 +15,8 @@ def gp_minimize(func, dimensions, base_estimator=None,
                 n_calls=100, n_random_starts=10,
                 acq_func="EI", acq_optimizer="auto", x0=None, y0=None,
                 random_state=None, verbose=False, callback=None,
-                n_points=10000, n_restarts_optimizer=5, xi=0.01, kappa=1.96,
-                noise="gaussian", n_jobs=1):
+                n_points=10000, n_restarts_optimizer=20, xi=0.0, kappa=1.96,
+                noise="gaussian", rvs="uniform", n_spray_points=10, n_jobs=1):
```

Comment:
We should discuss on the default values of `acq_optimizer` and `rvs`. Not sure about exposing `rvs` either. Opinions?